lib/rust/ensogl/component/scroll-area/src/lib.rs

```diff
@@ -64,6 +69,19 @@ ensogl_core::define_endpoints! {
 // === Scroll Area ===
 // ===================
 
+/// A mask for clipping the scroll area content.
+mod mask {
+    use super::*;
+    ensogl_core::define_shape_system! {
+        (style:Style, corner_radius: f32) {
+            let width: Var<Pixels> = "input_size.x".into();
+            let height: Var<Pixels> = "input_size.y".into();
+            let color = color::Rgba::new(1.0, 1.0, 1.0, 1.0);
```

Comment:
there is a function `white` or something for that.

Reply:
Done